specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json

```diff
@@ -1,21 +1,20 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2018-01-01-preview",
-    "title": "schemaregistry",
-    "description": "Description of the new service",
```

Comment:
I believe this description is used as the client description for code generation. So, it would be good to have something.

Reply:
Added description